src/wallet/electrum.cpp

```diff
@@ -484,7 +484,7 @@ static cpp_int mnemonic_decode(const word_list& mnemonic,
         if (position == -1)
             return { 1 };
 
-        entropy *= dictionary_size + position;
+        entropy = (entropy * dictionary_size) + position;
```

Comment:
I thought that this might have been the intent. Almost made this change, but obv that felt sketchy.